microsoft_teams/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: microsoft_teams
 title: Microsoft Teams
 description: The Microsoft Teams plugin allows you to send and trigger workflows on new messages. The plugin will also allow for teams management with the ability to add and remove teams, channels, and users
-version: 2.0.5
+version: 2.1.0
```

Comment:
This version and the version in bin are mismatched run make regenerate. Also as we are adding a new input to an existing action this will need to be a major bump

Reply:
Regenerated! thanks. Don't need major version bump for new optional inputs